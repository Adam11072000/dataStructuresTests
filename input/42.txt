init 150
addImage 180
setLabelScore 65 148 48 62
getHighestScoredLabel 180 36
getHighestScoredLabel 180 12
mergeSuperPixels 180 21 0
resetLabelScore 0 124 21
addImage 128
setLabelScore 180 23 219 190
addImage 23
resetLabelScore 74 -8 209
addImage 128
resetLabelScore 180 72 0
quit