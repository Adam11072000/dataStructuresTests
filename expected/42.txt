init done.
addImage: SUCCESS
setLabelScore: FAILURE
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
setLabelScore: SUCCESS
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: FAILURE
resetLabelScore: INVALID_INPUT
quit done.
